bin/travis.sh

```diff
@@ -4,7 +4,8 @@
 if [ "$1" == 'before' ]; then
 	cd "$WP_CORE_DIR/wp-content/plugins/woocommerce-admin/"
 	npm run build:feature-config
-	if [[ "$RUN_PHPCS" == "1" || "$RUN_RANDOM" == "1" ]]; then
+	# Load PHPUnit version in dev dependencies for PHP ^7.2 until WP unit tests support PHPUnit 8.X
+	if [[ "$COMPOSE_DEV" == "1" ]]; then
```

Comment:
Aside from a potential reduction in execution time, is there a reason to skip the dev dependencies in CI?

Nitpick: why "compose" dev and not "composer" dev?

Reply:
> is there a reason to skip the dev dependencies in CI?

PHPUnit 7 requires PHP 7.1 or higher

```
This version of PHPUnit is supported on PHP 7.1, PHP 7.2, and PHP 7.3.
You are using PHP 7.0.25 (/home/travis/.phpenv/versions/7.0.25/bin/php).
```

Looking at WC, the way they have implemented it is https://github.com/woocommerce/woocommerce/blob/master/.travis.yml#L71 . Would it be better to do it through allowing both versions in composer.json & always install dev?

> why "compose" dev and not "composer" dev?

Updated.